configs/bundle_config.yaml

```diff
@@ -396,3 +317,12 @@ databundles:
     output: [cutouts/cutout-2013-era5.nc]
     disable_by_opt:
       build_cutout: [all]
+
+  # data bundle containing the protected data for the whole world
+  bundle_landcover_earth:
+    countries: [Earth]
+    category: landcover
+    destination: "data/landcover/world_protected_areas"
+    urls:
+      protectedplanet: https://d1gam3xoknrgr2.cloudfront.net/current/WDPA_{month:s}{year:d}_Public_shp.zip
+    output: [data/landcover/world_protected_areas/*]
```

Comment:
Would you kill me if I'd suggest a hard-coded url instead of the dynamically-changed one? It looks like UN is a link to the latest copy of the WDPA shape file under http://wcmc.io/wdpa_current_release

(found [here](https://github.com/prioritizr/wdpar/blob/ce28c41b82b30fe5604b6452bd9c74f88325bdd3/R/wdpa_url.R#L114-L117))

Reply:
I was not aware of that link, we must do that!
I'll revert the latest changes

Reply:
Mmm I've investigated the database, however, the format of that download is different with respect to the usual zip file: the format is a gdb multi-layer data and does not contain shapefiles as for the standard approach.
Revising the approach to support gdb files is possible, but it seems an overkill in my opinion, what do you think?

Reply:
Sorry, I have been not aware of the differences in formats. Thanks for checking! 

I don't think that it makes sense to revise the approach for `gdb` files. We still have a number of input-validation issue which are more meaningful from my perspective (alternative hydrobasins, new natura.tiff...). Not sure it's a good idea to add points to this list :)

To me, the implemented approach looks perfectly fine. I have been just amazed by a simplicity of UN url 🙃 But if it doesn't going to work smoothly, I don't see any serious advantages there as compared to the solution you have still implemented.

Reply:
Super :) we are on the same page; I'm running locally a workflow to verify it works for the world. I'll soon push the (little) changes

Reply:
I've pushed the changes but the workflow is still running, tomorrow I'll check the status :)

Reply:
I confirm the successful execution of the workflow :) [it is stuck at building natura, but that's natural...]

Reply:
Super! :D